hazelcast-client/src/test/java/com/hazelcast/client/map/ClientMapBasicTest.java

```diff
@@ -54,13 +54,23 @@
 @RunWith(HazelcastParallelClassRunner.class)
 @Category(QuickTest.class)
 public class ClientMapBasicTest {
+    private static final String KEY = "key";
+    private static final String OLD_VALUE = "oldValue";
+    private static final String NEW_VALUE = "newValue";
 
-    static HazelcastInstance client;
-    static HazelcastInstance server;
+    private static HazelcastInstance client;
+    private static HazelcastInstance server1;
+    private static HazelcastInstance server2;
+    private static HazelcastInstance server3;
+
+    private String randomMapName = randomString();
```

Comment:
static vs. non-static. It is perfectly fine to do this, the non-static values are not shared between parallel test executions. I just printed the `randomMapName` and the test name to proof this:
```
04edd77a-f8f8-4e34-a3a8-aa50fa1b311d testPutTTL_AfterExpireWhenKeyExists()
06ff1c91-aa02-44d2-93c0-35f26d43be19 testPutTTL()
0d9c3228-2403-4e6a-b160-5bd4cbf6a731 testPut_whenValueNull()
1700a619-28b0-44a6-af3c-a49aa046dd59 testSize_withMultiKeyPuts()
2881be77-bf12-4032-b8c4-c5aac877297a testPutAsync_withKeyNull()
34747c38-ce57-4fcc-8af2-18a607878ecd testIsEmpty_afterPutRemove()
4f4fd00a-9264-4bda-83f8-28fcfc066841 testPut_whenKeyNull()
50d575d3-3539-4829-a3e7-0e00e62810d3 testSize_whenEmpty()
52993fa8-db22-402d-9420-8313615739ae testPutAsync()
5ce06c98-d082-493a-8a86-aab9307913fe testPutTTL_whenKeyExists()
80095c25-8c65-47b7-b609-39b6fd45fca0 testPutAsync_whenKeyExists()
99d7a8ed-1153-42f1-973a-ec008db6a1c2 testClientGetMap()
9be24525-d9c1-42d8-928c-f1c8c4874df3 testGetName()
a0c14388-a125-4480-b8be-412376b48f0f testPut_whenKeyExists()
a2e3a601-06f2-4beb-8c02-792ad6ffb050 testPutAsyncTTL()
bbb9b9db-5043-49f1-bdf7-b4c731410c70 testIsEmpty_whenNotEmpty()
ca222e9a-4e55-4708-94c9-3044df863ca8 testPutAsync_withValueNull()
d246dfc3-5606-4bd2-9982-ee32db0505e5 testPutTTL_AfterExpire()
e6ae214f-90e5-4987-9ed4-fc42195a4829 testPut()
e94cedb0-3442-455f-86f8-d50b65eef09d testIsEmpty_whenEmpty()
fe6126ce-0c1b-4d14-a264-0e28a338c710 testSize()
```
If it would not be the case the whole test could not become green, which it does.